submissions/adeyemiJuwon/database/init.sql

```diff
@@ -0,0 +1,222 @@
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `admin_users`
+--
+
+DROP TABLE IF EXISTS `admin_users`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `admin_users` (
+  `admin_user_id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(50) COLLATE utf8mb4_general_ci NOT NULL,
+  `email_address` varchar(250) COLLATE utf8mb4_general_ci NOT NULL,
+  `password` varchar(250) COLLATE utf8mb4_general_ci NOT NULL,
+  `phone` varchar(50) COLLATE utf8mb4_general_ci DEFAULT NULL,
+  PRIMARY KEY (`admin_user_id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `admin_users`
+--
+
+LOCK TABLES `admin_users` WRITE;
+/*!40000 ALTER TABLE `admin_users` DISABLE KEYS */;
+INSERT INTO `admin_users` VALUES (1,'juwon','samju7778@gmail.com','ade100','23480-667-76598'),
+(2,'samuel','samju6677@gmail.com','sam100','23480-667-76598'),
+(3,'nuru','nuru@gmail.com','nuru100','23480-667-87598'),(4,'itunu','itunu@gmail.com','itunu100','23480-667-66598'),
+(5,'feyi','feyi@gmail.com','feyi100','23480-567-76598'),
+(6,'temidayo','temi@gmail.com','temi100','23480-667-76598'),
+(7,'seye','seye@gmail.com','seye100','23480-667-76690');
+/*!40000 ALTER TABLE `admin_users` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `categorys`
+--
+
+DROP TABLE IF EXISTS `categorys`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `categorys` (
+  `category_id` int(11) NOT NULL AUTO_INCREMENT,
+  `category` varchar(100) COLLATE utf8mb4_general_ci NOT NULL,
+  PRIMARY KEY (`category_id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `categorys`
+--
+
+LOCK TABLES `categorys` WRITE;
+/*!40000 ALTER TABLE `categorys` DISABLE KEYS */;
+INSERT INTO `categorys` VALUES (1,'Laptop Accessiories '),
+(2,'Baby Care & Wear'),
+(3,'Women Collection'),
+(4,'Electronic Accessiories'),
+(5,'Smart Home & Garden'),
+(6,'Lights & Lighting');
+/*!40000 ALTER TABLE `categorys` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `customers`
+--
+
+DROP TABLE IF EXISTS `customers`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `customers` (
+  `customer_id` int(11) NOT NULL AUTO_INCREMENT,
+  `first_name` varchar(50) COLLATE utf8mb4_general_ci NOT NULL,
```

Comment:
Why are you setting this ` COLLATE utf8mb4_general_ci` in all the columns on this table?

Reply:
@adaugochi . Okay Done Thank you